site/src/store/slices/coursebagSlice.ts

```diff
@@ -12,7 +12,14 @@ export const coursebagSlice = createSlice({
       state.coursebag = action.payload;
```

Comment:
note that because this sets the initial state, you will need to sort the payload here. Consider adding a helper function that compares course order, then use that both here and in the `addCourseToBagState` function